webpack/assets/javascripts/react_app/components/common/TimeAgo.js

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import Moment from 'react-moment';
+
+export default ({ date }) => date && <Moment fromNow={true} parse="YYYY-MM-DD h:mm:s z" date={date} /> || <span>{__('N/A')}</span>;
```

Comment:
does i18n works as expected? /cc @tstrachota

Reply:
It does not. Moment needs special treatment.

Reply:
/cc @tstrachota as he is working on date time display in react too.